package.json

```diff
@@ -29,25 +29,27 @@
   },
   "homepage": "https://github.com/syuraj/nepaltoday-api#readme",
   "dependencies": {
+    "@babel/cli": "^7.6.0",
```

Comment:
We need to find a way to transpile `es6` to `es5` and not require `babel` in production built.